src/main/java/com/spikes2212/dashboard/AutoChooser.java

```diff
@@ -0,0 +1,144 @@
+     * @param defaultOption the default command this {@link SendableChooser} will use as the default option
+     * @param options the commands that will be added to this {@link SendableChooser} other than the default command
+     */
+    public AutoChooser(Command defaultOption, Command... options) {
+        this(DEFAULT_NAMESPACE_NAME, defaultOption, options);
+    }
+
+    /**
+     * Creates an autonomous chooser where the first {@link Command} and {@link String} will be used as the default
+     * option, and the rest of the options will be parsed as following: every even index (0,2,4,...) will be a command
+     * and every odd index (1,3,5,...) will be used as the name for the previous command.
+     *
+     * @param rootNamespaceName the name that will be given to the {@link RootNamespace}
+     * @param defaultOption the default command this {@link SendableChooser} will use as the default option
+     * @param defaultOptionName the name for the default command
+     * @param options the rest of the options, where every even index is a command to be added and every odd index is
+     *               the name for the previously mentioned command
+     */
+    public AutoChooser(String rootNamespaceName, Command defaultOption, String defaultOptionName, Object... options) {
+        rootNamespace = new RootNamespace(rootNamespaceName);
+        setDefaultOption(defaultOptionName, defaultOption);
+        Command command;
+        String name;
+        for (int i = 0; i < options.length - options.length % 2; i += 2) {
+            if (options[i] instanceof Command) {
+                command = (Command) options[i];
+            } else {
+                throw new IllegalArgumentException("The " + (i + 3) + " argument is not a Command.");
+            }
+            if (options[i + 1] instanceof String) {
+                name = (String) options[i + 1];
+            } else {
+                throw new IllegalArgumentException("The " + (i + 4) + " argument is not a String.");
+            }
+            addOption(name, command);
+        }
+        putOnShuffleboard();
+    }
+
+    /**
+     * Creates an autonomous chooser where the first {@link Command} and {@link String} will be used as the default
+     * option, and the rest of the options will be parsed as following: every even index (0,2,4,...) will be a command
+     * and every odd index (1,3,5,...) will be used as the name for the previous command.
+     *
+     * @param defaultOption the default command this {@link SendableChooser} will use as the default option
+     * @param defaultOptionName the name for the default command
+     * @param options the rest of the options, where every even index is a command to be added and every odd index is
+     *               the name for the previous command
+     */
+    public AutoChooser(Command defaultOption, String defaultOptionName, Object... options) {
+        this(DEFAULT_NAMESPACE_NAME, defaultOption, defaultOptionName, options);
```

Comment:
make the indices printed correct (either by breaking the `this` call or by making a private constructor both this constructor and the previous will use as a helper method that receives the incrementation amount as a parameter)